ogcapi-features/README.md

```diff
@@ -1,42 +1,93 @@
+  - [STAC API - Collections](openapi-collections.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-rc.1/collections))
+- **Conformance Class URIs:**
+  - <https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features> - Features
+  - <https://api.stacspec.org/v1.0.0-rc.1/collections> - Collections
+  - <http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core> - [OGC API - Features - Conformance Class Core](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#rc_core)
+  - <http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson> - [OGC API - Features - Conformance Class GeoJSON](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_requirements_class_geojson)
+  - <http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/oas30> - [OGC API - Features - Conformance Class OpenAPI 3.0](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#rc_oas30)
 - **[Maturity Classification](../README.md#maturity-classification):** Candidate
 - **Dependencies**:
   - [STAC API - Core](../core)
-  - [STAC API - Collections](../collections)
   - [OGC API - Features](https://www.ogc.org/standards/ogcapi-features)
   uses all the OGC API - Features openapi fragments to describe returning STAC Item objects.
 
-Adding OGC API - Features (OAFeat) to a STAC API means fully implementing all its requirements, and then returning STAC
-[Item](../stac-spec/item-spec/README.md) objects from its `/items` endpoints. In OAFeat, GeoJSON is an optional
-conformance class, enabling flexibility. However, STAC requires the use of GeoJSON for OAFeat
-endpoints.
+The *STAC API - Features and Collections* specification extends the
+[OGC API - Features - Part 1: Core](https://docs.opengeospatial.org/is/17-069r3/17-069r3.html)
+(OAFeat) specification. OAFeat
+allows returning any content type from its endpoints, whereas *STAC API - Features* requires support
+for returning STAC [Item](../stac-spec/item-spec/README.md) and STAC [Collection](../stac-spec/item-spec/README.md)
+entities. As these entities are also GeoJSON types, the GeoJSON conformance class of OGC API - Features is supported
+(<http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson>).
+
+## Conformance Classes
+
+While OAFeat defines a single conformance class for its endpoints, STAC API divides these behaviors into two
+conformance classes -- Features and Collections. The STAC API - Features
+(<https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features>) conformance class includes and extends the behavior
+of OAFeat, while the STAC API - Collections (<https://api.stacspec.org/v1.0.0-rc.1/collections>) conformance
+class is the subset of Features that pertains only to Collections.
+
+### STAC API - Features
+
+The *STAC API - Features* (<https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features>) conformance class
+encompasses all of the behavior described in this specification, as
+derived from OAFeat.
+
+### STAC API - Collections
```

Comment:
Throughout the document Features is always first and then follow Collections. In my head I'd logically expect Collections first and then Features, because Collections is basically the first step you'd implement and then the additions from Features. Thoughts?

Reply:
Collections is a subset of the behavior of Features. You're not required to list collections, but you might as well do it. 

For a client that's looking to use the /collections endpoint, they should look for either one one of them.

Reply:
updated the order of Collections and Features.